src/pages/home/report/ReactionList/PopoverReactionList/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React, {forwardRef, useImperativeHandle, useRef, useState} from 'react';
+import type {ForwardedRef} from 'react';
+import type {InnerReactionListRefType} from '@hooks/useBasePopoverReactionList/types';
+import type {ReactionListRef} from '@pages/home/ReportScreenContext';
+import BasePopoverReactionList from './BasePopoverReactionList';
+
+type PopoverReactionListProps = {
+    ref: ForwardedRef<ReactionListRef>;
+};
+
+function PopoverReactionList(props: PopoverReactionListProps) {
```

Comment:
Refer to React forwardRef [docs](https://react.dev/reference/react/forwardRef). ref should be the second parameter here.